skpy/chat.py

```diff
@@ -492,27 +504,29 @@ def chat(self, id):
                                auth=SkypeConnection.Auth.RegToken, params={"view": "msnp24Equivalent"}).json()
         return self.merge(SkypeChat.fromRaw(self.skype, json))
 
-    def create(self, members=(), admins=()):
+    def create(self, members=(), admins=(), moderated=False):
         """
         Create a new group chat with the given users.
 
         The current user is automatically added to the conversation as an admin.  Any other admin identifiers must also
-        be present in the member list.
+        be present in the member list. You can also make the chat moderated by passing moderated value
```

Comment:
This should just be stated in the argument description below, which currently doesn't describe its use.